capstone-frontend/src/styles/ListPage.css

```diff
@@ -0,0 +1,7 @@
+.light-gray-border {
+  border: 1px solid #ccc;
+}
+
+.width-75 {
+  width: 75%;
+}
```

Comment:
nit: Add a newline here so this doesn't get caught in the diff if you add more rules below.

Reply:
Ok.